pkgs/applications/editors/neovim/tests/default.nix

```diff
@@ -150,7 +150,6 @@ rec {
     echo '\documentclass{article}' > main.tex
 
     ${nvim_with_ftplugin}/bin/nvim main.tex -c "set ft?" -c quit
-    ls -l $TMPDIR
```

Comment:
can we keep this one ? It's helpful to debug a failing test.

Reply:
maybe `echo "TMPDIR: $TMPDIR"` is enough?

Reply:
I am probably one of the few people building this package and I am a dumb person, it helps me to see what files are generated to check if the test hasn't gone awfully wrong.

> found some ls execution in packages.

why is that an issue ?

Reply:
not really. As you said it is often used for debugging and forgotten. We keep it. All fine.